utils/dd_to_csv.py

```diff
@@ -201,12 +200,13 @@ def convert_dd_to_tabular(
     os.makedirs(set_path, exist_ok=True)
 
     # Extract headers with key=param_name and value=List[attributes]
-    lines = list(open("xl2times/config/times_mapping.txt", "r"))
+    lines = list(open("xl2times/config/times_mapping.txt"))
     headers_data = headers_by_attr
     # The following will overwrite data obtained from headers_by_attr
     # TODO: Remove once migration is done?
     for line in lines:
-        line = line.strip()
+        # TODO Fix: PLW2901 `for` loop variable `line` overwritten by assignment target
+        line = line.strip()  # noqa
```

Comment:
One way to fix this is to change the loop to `for line in (l.strip() for l in lines):`